### ISSUE TITLE
Scroller widget + pager addon

### ISSUE THREAD

**Issue by xikky** [Mottie/tablesorter#884]
I need a paginated table with a fixed column to the left, but a conflict is occuring. On pressing the next page button, all columns change but not the fixed one.

I have a forked fiddle with the code to show the problem. Please check it out in the following link.
https://jsfiddle.net/atyh8zuu/

**Comment by Mottie** [comment]
Hi @xikky!

For now, try this workaround until I get some time to update the widget:

```js
$('table').on('pagerComplete', function(){
    // refresh the fixed columns in the scroller
    $( this ).trigger( 'setFixedColumnSize' );
});
```

**Comment by xikky** [comment]
Great, will be able to check it out on monday. Thanks!

**Comment by xikky** [comment]
Thanks Mottie it works fine now. My only problem now is that I have like a blank sticky row on top and the fixed column height is not equal to the rest of the row.

![tblscroller_sticky_row](https://cloud.githubusercontent.com/assets/8024321/7345002/50f73b5a-ecd1-11e4-8750-20afaf171069.png)

Any ideas why this might be happening?

**Comment by Mottie** [comment]
To make the fixed column show the correct number of rows, set the pager [`removeRows` option](http://mottie.github.io/tablesorter/docs/#pager-removerows) to `true`.

I was working on this last night and I just could not get the columns to line up. I'll need to add some additional code tweaks to maybe remove the `colgroup` and figure out what else is interfering.

Oh, also, you'll need to include the `pagerInitialized` event for the update:

```js
$('table').on('pagerComplete pagerInitialized', function(){
    // refresh the fixed columns in the scroller
    $( this ).trigger( 'setFixedColumnSize' );
});
```

The above addition will no longer be necessary once I get this issue worked out.

**Comment by xikky** [comment]
Thanks for your interest in this issue. I had encountered a couple of conflicts between fixed column and pager, which I couldn't give more time to find work arounds at work so for the moment I just removed the fixed column functionality. Moreover I have an empty table which is populated on click of a search button through Ajax calls, and this makes things a bit more complicated. But I would love to hear about any new updates on this matter.
